README.md

```diff
@@ -22,6 +22,95 @@ However, as noted above we now recommend installing and loading `ITensorMPS` ins
 
 ## News
 
+### ITensorTDVP.jl v0.4 Release Notes
+
+#### Breaking changes
+
+- When calling `tdvp(operator, t, init; kwargs...)`, `t` is now interpreted as the total evolution time, while in `ITensorTDVP.jl` v0.3 and below it was being interpreted as the time step. To upgrade, change code like:
+```julia
+tdvp(operator, -0.1im, init; nsweeps=10, kwargs...)
+```
+to:
+```julia
+tdvp(operator, -1.0im, init; nsweeps=10, kwargs...)
+```
+to evolve to time `1.0` over `10` steps with time steps of `0.1`. Also note that `ITensorTDVP.jl` v0.4 introduces `nsteps` as an alias for `nsweeps` in the `tdvp` function. `nsteps` is now the preferred syntax for specifying the number of time steps, and `nsweeps` may be deprecated in `tdvp` in the future. For example:
```

Comment:
```suggestion
tdvp(operator, -1.0im, init; nsweeps=10, kwargs...)
```
to evolve to total time `1.0` over `10` time steps of size `0.1`. Also note that `ITensorTDVP.jl` v0.4 introduces `nsteps` as an alias for `nsweeps` in the `tdvp` function. `nsteps` is now the preferred syntax for specifying the number of time steps, and `nsweeps` may be deprecated in `tdvp` in the future. For example:
```